libmodplug/VITABUILD

```diff
@@ -0,0 +1,23 @@
+pkgname=libmodplug
+pkgver=0.8.8.5
+pkgrel=1
+url="http://modplug-xmms.sourceforge.net/"
```

Comment:
why not use github [repo](https://github.com/Konstanty/libmodplug)?

Reply:
I didn't notice that libmodplug was still maintained on github by its author. The only boring thing is that he doesn't tag releases. 
I'm ok to use it, but should I consider the last commit on master branch as the last revision ?